src/framework/uicomponents/qml/MuseScore/UiComponents/TextInputField.qml

```diff
@@ -182,9 +182,10 @@ FocusScope {
                     return
                 }
 
-                if (event.key === Qt.Key_Enter || event.key === Qt.Key_Return
-                        || event.key === Qt.Key_Escape) {
+                if (event.key === Qt.Key_Enter || event.key === Qt.Key_Return || event.key === Qt.Key_Escape) {
                     event.accepted = true
+                    // In case one of these keys were user-defined as command:
+                    textInputFieldModel.handleShortcut(event.key, event.modifiers);
```

Comment:
This is the best spot I could find at the time to allow for [enter], if that were to be the user-defined key sequence for this command,  to be used while in active palette-search edit